easybuild/easyconfigs/l/LS-PrePost/LS-PrePost-4.7.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "Tarball"
+
+name = 'LS-PrePost'
+version = '4.7'
```

Comment:
Looks like they have started to use all 3 numbers for versions, which would solve problem with different date suffixes for the same "version". I think we should use 4.7.8 going forward.

Reply:
ok, will fix it, let's hope that they wont change dates anymore